<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Path_L">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>289</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Путь</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Logint_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>289</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>289</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="App_L">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>130</y>
      <width>289</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Приложение/сайт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Review_B">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateLogin_B">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенероровать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GeneratPassword_B">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенероровать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_serch_B">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить/найти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
